src/main/java/seedu/mindmymoney/helper/Calculations.java

```diff
@@ -0,0 +1,34 @@
+package seedu.mindmymoney.helper;
+
+import seedu.mindmymoney.MindMyMoneyException;
+import seedu.mindmymoney.data.ExpenditureList;
+import seedu.mindmymoney.userfinancial.Expenditure;
+
+import java.util.ArrayList;
+
+import static seedu.mindmymoney.constants.ExpenditureFields.TIME;
+import static seedu.mindmymoney.helper.GeneralFunctions.findItemsInList;
+
+/**
+ * Container for functions that help do calculations.
+ */
+public class Calculations {
+
+    /**
+     * Calculates the total expenditure in a given month.
+     *
+     * @param input The month to calculate expenditure for.
+     * @param expenditureList The list containing all expenditures to search for.
+     * @throws MindMyMoneyException when findItemsInList throws MindMyMoneyException.
+     */
+    public static void calculateExpenditurePerMonth(String input, ExpenditureList expenditureList)
+            throws MindMyMoneyException {
+        ArrayList<Expenditure> foundItems = findItemsInList(input, TIME.toString(), expenditureList);
+        int sumOfExpenditure = 0;
+        for (Expenditure item : foundItems) {
+            sumOfExpenditure += item.getAmount();
+        }
+        System.out.println("Total expenditure in the month of " + input + " is " + sumOfExpenditure);
+    }
+
+}
```

Comment:
Consider returning the total expenditure instead, and have the caller deal with printing the result, to improve SLAP.

Reply:
hmm, ok will consider!